packages/net-misc/remmina/remmina.exlib

```diff
@@ -12,21 +12,30 @@ Remmina is a remote desktop client written in GTK+, aiming to be useful for syst
 HOMEPAGE+=" http://remmina.org/"
 
 LICENCES="GPL-2"
-MYOPTIONS=""
+MYOPTIONS="ssh rdp vnc"
```

Comment:
Why do we introduce these options? You have a usecase to disable any of them?

Reply:
Sure, let me explain it.

Personally I don't need an SSH client in this application, and especially when it triggers dependency on libvte. I see VTE-based terminals as pretty basic and slow en masse, so why keep (and periodically update, etc.) reduntant single-dependent package for unused feature?
I am using Remmina only as RDP client. But of course I can presume that others don't want this feature with it's dependency installed (as SSH for me).
And VNC done in the same way just for uniformity.

Reply:
> And VNC done in the same way just for uniformity.

I don't know what that means. So you have a use case for disabling `ssh`, the other options then are not really needed afais. Keep in mind that options always increase complexity (of configuration, maintenance, ...)

Reply:
> I don't know what that means.
I was trying to say that if allow to choose single protocol then choosing others too seems to be reasonable. (Actually I have no need in VNC too and disabled it after testing build, but I bear so much personal antipathy only to libvte)

However I can understand your point about increasing complexity and will not insist if you don't wanna to have it here.

Reply:
So let's keep ssh only

Reply:
Fine, I've ammended adjusted version.